snooty.toml

```diff
@@ -1,5 +1,5 @@
 name = "node"
-toc_landing_pages = ["/crud", "/crud/read-operations", "/crud/write-operations"]
+#toc_landing_pages = ["/crud", "/crud/read-operations", "/crud/write-operations"]
```

Comment:
should this '#' be removed? I think this line was accidentally commented out

Reply:
It was intentionally commented out. I was debugging a build issue and Andrew Aldridge mentioned this is not yet supported and asked me to comment it out until they officially support it.

Reply:
Ah I understand, thanks for the clarification